src/renderer/components/+workloads-pods/pods.tsx

```diff
@@ -92,7 +94,7 @@ export class Pods extends React.Component<Props> {
         renderHeaderTitle={<Trans>Pods</Trans>}
         renderTableHeader={[
           { title: <Trans>Name</Trans>, className: "name", sortBy: sortBy.name },
-          { className: "warning" },
+          { className: "warning", showWithColumn: "name" },
```

Comment:
Why it must be shown only with name? I am agaisnt the decision of adding such a property. It adds complexity and overall show/hide TableCell behavior becomes not so straightforward for devs and consumers. Imho, all columns must be showed/hidden by hitting according checkbox only.

Reply:
This this still an issue for you Aleks? I think this also ties into not relying on the class names.

Reply:
Not an issue anymore. Let's stick with current behavior.